src/main/kotlin/nl/avisi/structurizr/site/generatr/site/model/DiagramViewModel.kt

```diff
@@ -1,19 +1,29 @@
 package nl.avisi.structurizr.site.generatr.site.model
 
 import com.structurizr.view.View
+import nl.avisi.structurizr.site.generatr.site.GeneratorContext
+import java.io.File
 
 data class DiagramViewModel(
     val name: String,
+    val svg: String,
     val svgLocation: ImageViewModel,
     val pngLocation: ImageViewModel,
     val pumlLocation: ImageViewModel
 ) {
     companion object {
         fun forView(pageViewModel: PageViewModel, view: View) = DiagramViewModel(
             view.name,
+            File(svgPathname(pageViewModel.generatorContext, view))
+                .let {
+                    if (it.exists()) it.readText() else "<svg></svg>"
```

Comment:
Having the fallback here makes unit testing much easier. We could alternatively use something like `File * has not been found` Instead of `<svg></svg>` to at least give an hint.

Reply:
This is no longer the case.